scene/gui/color_picker.h

```diff
@@ -83,14 +98,18 @@ class ColorPicker : public BoxContainer {
 	Label *labels[4];
 	Button *text_type = memnew(Button);
 	LineEdit *c_text = memnew(LineEdit);
+
 	bool edit_alpha = true;
 	Size2i ms;
 	bool text_is_constructor = false;
-	int presets_per_row = 0;
 	PickerShapeType picker_type = SHAPE_HSV_WHEEL;
 
+	int preset_column_count = 9;
```

Comment:
```suggestion
	const int preset_column_count = 9;
```
It could be constant, it's not modified anywhere.

Reply:
Done.